packages/ui/src/views/layouts/SandboxLayout.tsx

```diff
@@ -10,44 +10,26 @@ interface ColumnsProps {
 
 function Root({ children }: { children: ReactNode }) {
   return (
-    <main className="h-screen" role="main">
+    <main className="grid min-h-screen grid-cols-[auto_1fr]" role="main">
       {children}
     </main>
   )
 }
 
 function LeftPanel({ children, className }: { children: ReactNode; className?: string }) {
   return (
-    <nav
-      className={cn('border-borders-5 fixed bottom-0 left-0 top-0 z-50 w-[220px] overflow-y-auto border-r', className)}
-      aria-label="Left Navigation Panel"
-    >
-      {children}
-    </nav>
+    <div className="sticky top-0 h-screen">
+      <nav className={cn('h-full w-56 border-borders-5 border-r', className)} aria-label="Left Navigation Panel">
```

Comment:
`w-56` is not correct. we want nav to be exactly 220px, and that should be controlled by the grid layout on line 13

Reply:
Done